com.unity.render-pipelines.high-definition/Documentation~/Upgrading-from-2021.1-to-2021.2.md

```diff
@@ -93,3 +93,7 @@ When the **Dynamic Render Pass Culling** option is enabled in the HDRP Global Se
 ## Dynamic Resolution
 
 From 2021.2, Bilinear and Lanczos upscale filters have been removed as they are mostly redundant with other better options. A project using Bilinear filter will migrate to use Catmull-Rom, if using Lanczos it will migrate to Contrast Adaptive Sharpening (CAS).  If your project was relying on those filters also consider the newly added filters TAA Upscale and FidelityFX Super Resolution 1.0.
+
+## Ambient Mode
+
+In order to offer a better first experience to users, the Ambient Mode parameter in the Visual Environment volume component now defaults to Dynamic. This may impact existing projects where no default volume profile overriding this parameter exists. In order to return to the former behavior, you need to add a Visual Environment component to the default volume profile and override the Ambient Mode to static.
```

Comment:
"override the Ambient Mode to static" - How is this actually done by the user?

Reply:
This is done on the Visual Environment VolumeComponent.

Reply:
What does the user have to click on to get to it?

Reply:
It's in this component https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@12.0/manual/Override-Visual-Environment.html which could be potentially in a Volume local to a scene or the default volume profile contained in the Global HDRP settings.
For reference, volume framework doc: https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@12.0/manual/Volumes.html

Reply:
Good info gathering! I'm just going to make some language adjustments to your suggestion :)